notifications/rest/notifications/send-notification-detailed/send-notification-detailed.3.x.js

```diff
@@ -11,14 +11,26 @@ var service = client.notify.services('ISxxx');
 
 service.notifications
   .create({
-      identity:'00000001',
-      title:'Generic loooooooong title for all Bindings',
-      body:'This is the body for all Bindings',
-      fcm:'{"notification":{"title":"New alert","body":"Hello Bob!"}}',
-      apn:'{"aps":{"alert":{"title":"New alert","body":Hello Bob!"}}}',
+    "identity": "00000001",
+    "title": "Generic loooooooong title for all Bindings",
+    "body": "This is the body for all Bindings",
+    "fcm": {
+      "notification": {
+        "title": "New alert",
+        "body": "Hello Bob!"
+      }
+    },
+    "apn": {
+      "aps": {
+        "alert": {
+          "title": "New alert",
+          "body": "Hello Bob!"
+        }
+      }
+    },
   }).then(function(notification) {
-      console.log(notification);
+    console.log(notification);
   }).catch(function(error) {
```

Comment:
perhaps this one should not catch the error and let the snippet tester handle it instead?